emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java

```diff
@@ -33,44 +34,54 @@
 
   private RecentEmojiGridView recentEmojiGridView;
 
-  EmojiPagerAdapter(final OnEmojiClickListener listener,
-                    final OnEmojiLongClickListener longListener,
-                    final RecentEmoji recentEmoji, final VariantEmoji variantManager) {
+  EmojiPagerAdapter(
+      final OnEmojiClickListener listener,
+      final OnEmojiLongClickListener longListener,
+      final RecentEmoji recentEmoji,
+      final VariantEmoji variantManager
+  ) {
     this.listener = listener;
     this.longListener = longListener;
     this.recentEmoji = recentEmoji;
     this.variantManager = variantManager;
-    this.recentEmojiGridView = null;
+  }
+
+  boolean hasRecentEmoji() {
+    return !(recentEmoji instanceof NoRecentEmoji);
+  }
+
+  int recentCount() {
```

Comment:
This name confused me, I thought this means how many recent emojis the user has saved from using the component previously. Maybe inline this in the `EmojiView` constructor or find a different name?

Reply:
Have to agree here, do we need this method?

Reply:
Yeah it's being used multiple times (also for the top bar, when we don't have that recent emoji tab)

Does anyone have a better name?